include/tvm/runtime/data_type.h

```diff
@@ -72,6 +73,9 @@ class DataType {
     data_.code = static_cast<uint8_t>(code);
     data_.bits = static_cast<uint8_t>(bits);
     data_.lanes = static_cast<uint16_t>(lanes);
+    if (code == kBFloat) {
+      CHECK_EQ(bits, 16);
```

Comment:
It is understandable that right now we only support bf16, but my concern is that "should we put the check here"?

Reply:
I understand your concern. Any suggestions for the location where we put this check? Thanks.

Reply:
@tqchen This is just a nitpick. What do you think?

Reply:
let us leave it as it is for now, we can come back to it later